app/src/middleware/requestSanitizer.ts

```diff
@@ -0,0 +1,50 @@
+import type { NextFunction, Request, Response } from 'express';
+
+/**
+ * Removes all NUL characters from a string
+ * @param str The string to sanitize
+ * @returns The sanitized string if it includes NUL character(s), otherwise the original string
+ */
+function stripNullCharactersFromString(str: string): string {
+  // eslint-disable-next-line no-control-regex
+  return str.replace(/\u0000/g, '');
```

Comment:
The eslint ignore should be one-lined when possible, similar to the no-console ones used periodically
```ts
return str.replace(/\u0000/g, ''); // eslint-disable-line no-control-regex
```